From: Alexey Dobriyan <adobriyan@gmail.com>
Subject: adfs: delete unused "union adfs_dirtail" definition
Date: Wed, 9 Aug 2023 19:35:49 +0300

union adfs_dirtail::new stands in the way if Linux++ project:
"new" can't be used as member's name because it is a keyword in C++.

Link: https://lkml.kernel.org/r/43b0a4c8-a7cf-4ab1-98f7-0f65c096f9e8@p183
Signed-off-by: Alexey Dobriyan <adobriyan@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 fs/adfs/dir_f.h |    5 -----
 1 file changed, 5 deletions(-)

--- a/fs/adfs/dir_f.h~adfs-delete-unused-union-adfs_dirtail-definition
+++ a/fs/adfs/dir_f.h
@@ -58,9 +58,4 @@ struct adfs_newdirtail {
 	__u8 dircheckbyte;
 } __attribute__((packed));
 
-union adfs_dirtail {
-	struct adfs_olddirtail old;
-	struct adfs_newdirtail new;
-};
-
 #endif
_
